eng/Version.Details.xml

```diff
@@ -215,9 +215,9 @@
       <Uri>https://github.com/dotnet/arcade</Uri>
       <Sha>7215d8265a7fbcd022eb72ff7a6e2048444c985f</Sha>
     </Dependency>
-    <Dependency Name="Microsoft.SourceBuild.Intermediate.source-build-reference-packages" Version="6.0.0-servicing.22158.2">
+    <Dependency Name="Microsoft.SourceBuild.Intermediate.source-build-reference-packages" Version="6.0.0-servicing.22154.3">
```

Comment:
This build is 4 days older than what we have right now, don't think we want this